### ISSUE TITLE
RSS/Feed for every page that uses Blog template

### ISSUE THREAD

**Issue by mynameisfreedom** [bnomei/kirby3-feed#62]
Hi @bnomei, thank you for the great plugin. 

It is possible to use it to add RSS/Feed for every page that uses a Blog template?
So, not for general website feed, but similar as content representation like Kirby does with JSON 

https://getkirby.com/docs/cookbook/content-representations/generating-json#content-representation-for-the-blog

**Comment by bnomei** [comment]
I am unsure if I understand your question entirely, but the plugin is quite flexible as it is only a `pagesMethod` calling a static helper `\Bnomei\Feed::feed` at its core. Which you could call on any pages collection and echo the result at any point.

https://github.com/bnomei/kirby3-feed/blob/1bb0078638f75395e3dbb64d010c6ae1aa252349/index.php#L25

**Comment by mynameisfreedom** [comment]
Hi @bnomei, thank you for your fast answer. I actually didn't check the plugin code at all, and I was just wondering if my needed scenario is maybe available in the plugin options but just isn't covered in the plugin information. 

Now when I checked it out thoroughly, I see it is the best solution for me to create [content representation](https://getkirby.com/docs/guide/templates/content-representations) (blog.rss.php) myself and use your [RSS snippet](https://github.com/bnomei/kirby3-feed/blob/master/snippets/feed/rss.php) example as a base and extend it further. And add some routing to convert blog.rss URL to blog/rss URL. 

That way I will have more flexibility and have a simpler solution for my case. One of my clients is using this plugin, I integrated it for him, and it works great. 

You are doing fantastic work, thank you! 🙏
